services/catarse/config/initializers/cookies_serializer.rb

```diff
@@ -4,4 +4,4 @@
 
 # Specify a serializer for the signed and encrypted cookie jars.
 # Valid options are :json, :marshal, and :hybrid.
-Rails.application.config.action_dispatch.cookies_serializer = :marshal
+Rails.application.config.action_dispatch.cookies_serializer = :hybrid
```

Comment:
Essa mudança é por esse problema aqui que herdamos do nosso Rails antigo:
https://bigbinary.com/blog/migrating-existing-session-cookies-while-upgrading-to-rails-4-1-and-above